app/javascript/role_manager.js

```diff
@@ -132,11 +149,43 @@ App.StimulusApp.register('role-manager', class extends Stimulus.Controller {
     this.setState(this.initialState)
   }
 
+  valueForTarget(t) {
+    return $(t).data('roleManagerRoleValue')
+  }
+
+  // enable columns that were previously enabled
+  setInitialColumns() {
```

Comment:
Seems like our state management isn't quite right given the code below has some code-smell. Probably fine for now though